src/LibraryUtilities.ts

```diff
@@ -371,11 +371,53 @@ export function buildLibrarySectionsFromLayoutSpecs(loadedTypes: any, layoutSpec
     let miscSection = sections.find(x => x.text == "Miscellaneous");
 
     results.push(convertToDefaultSection(typeListNodes, defaultSection));
+
+    _.each(sections, function (section) {
+        if (section.text != "default" && section.text != "Miscellaneous") {
+            results.push(convertToOtherSection(typeListNodes, section));
+        }
+    })
+
     results.push(convertToMiscSection(typeListNodes, miscSection));
 
     return results;
 }
 
+function convertToOtherSection(typeListNodes: TypeListNode[], section: LayoutElement): ItemData {
+    let sectionData = convertSectionToItemData(section);
+    let prefixes = section.include;
+    let nodeToProcess: TypeListNode[] = [];
+
+    _.each(typeListNodes, function (node) {
+        _.each(prefixes, function (prefix) {
+            if (node.fullyQualifiedName.startsWith(prefix.path)) {
+                let newName = (prefix.path.indexOf("://") == -1) ? node.fullyQualifiedName : node.fullyQualifiedName.split(prefix.path)[1];
+
+                // Copy all data over
+                let newTypeListNode = new TypeListNode(newName);
+                newTypeListNode.fullyQualifiedName = newName;
+                newTypeListNode.contextData = node.contextData;
+                newTypeListNode.iconUrl = node.iconUrl;
+                newTypeListNode.memberType = node.memberType;
+                newTypeListNode.processed = true;
```

Comment:
I am guessing that is because the needs of removing `pkg://` from the FQN. In that case can we do this:

```js
if (node.fullyQualifiedName.startsWith(prefix.path)) {
    let oldName = node.fullyQualifiedName;
    let tempName = node.fullyQualifiedName.split(prefix.path)[1];
    node.fullyQualifiedName = tempName;

    // Insert under section first
    buildLibraryItemsFromName(node, sectionData);

    // Then change back to the old name
    node.fullyQualifiedName = oldName;
}
```

Does that cause any problem?